### PR TITLE
CE-updates-patch-3.15.2

### PR DESCRIPTION
**Pull request by bmckercher123**
### Docs publishing updates 3.15.2

WAITING FOR RN REVIEW

- Component version updates
- Helm chart version
- Other doc updates

### CODE REVIEW

File: calico-enterprise_versioned_docs/version-3.15/variables.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const variables = {
   noderunning: 'calico-node',
   rootDirWindows: 'C:\\TigeraCalico',
   registry: 'quay.io/',
-  chart_version_name: 'v3.15.1-0',
+  chart_version_name: 'v3.16.2-0',
```
Comment (doucol) [inline calico-enterprise_versioned_docs/version-3.15/variables.js]: Should this be v3.15.2?

**Review by doucol — CHANGES_REQUESTED** [review]

File: calico-enterprise_versioned_docs/version-3.15/variables.js (hunk 1 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.15/variables.js]: Not according to Dan Fox.

Comment (rene-dekker) [reply calico-enterprise_versioned_docs/version-3.15/variables.js]: That must be a misunderstanding, our helm chart versions always match the major.minor.patch  that you are releasing.

Comment (rene-dekker) [reply calico-enterprise_versioned_docs/version-3.15/variables.js]: For reference, you can always look here: https://github.com/tigera/calico-private/blob/release-calient-v3.15/calico/_data/versions.yml

Comment (danudey) [reply calico-enterprise_versioned_docs/version-3.15/variables.js]: This was a typo on my end, sorry for the confusion! Yes, it should be 3.15.2 and not 3.16.2, sorry @bmckercher123!

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.1-release-notes.mdx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-27 Jan 2023
+xx Mar 2023
```
Comment (ctauchen) [inline calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.1-release-notes.mdx]: I think this change belongs in the other file, `_v3.15.2-release-notes.mdx`.

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+27 Jan 2023
```
Comment (ctauchen) [inline calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx]: XX March 2023 ...

**Review by ctauchen — COMMENTED** [review]
Just a quick note about a date change. Needs RN, but otherwise LGTM!

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx (hunk 3 above)
Comment (rene-dekker) [reply calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx]: Looks like this page is a copy of v3.15.1.

For an actual compilation of notes, check https://tigera.slack.com/archives/GS3MM6YR0/p1678311487947449

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.1-release-notes.mdx (hunk 2 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.1-release-notes.mdx]: Fixed

File: calico-enterprise_versioned_docs/version-3.15/variables.js (hunk 1 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.15/variables.js]: Fixed.

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+### Bug Fixes
+
+- Fixed a bug that caused policies to show in the service graph for even if they were deleted before the selected time range.
+- Fixed an issue where publicly-signed Splunk certificates were not trusted.
+- Fixed a bug that caused policy preview to crash due to a malformed query.
+- Fixed a bug that caused Anomaly Detection jobs to fail if incompletele documents are present in our log storage.
+
+### Known issues
+
+- L7 log collection fails to deploy on CIS hardened clusters. As a result, some cards in the manager UI dashboard will not display any metrics.
```
Comment (bmckercher123) [inline calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx:22]: I left these items from 3.15.1. Not sure if they are still issues.

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx [hunk 5]
```diff
@@ -0,0 +1,38 @@
+22 Mar 2023
+
+### Improvements
+
+- Added tolerations to CSI Driver so it can be scheduled on all nodes.
+- Adding graceful shutdown to Typha improves the "happy case" for upgrade, ensuring that shutting down a Typha instance doesn't cause a thundering herd of `calico-node` subcomponent restarts. Instead, clients are disconnected slowly over a configured interval, thus limiting the rate of `calico-node` subcomponent restarts.
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx]: I think this is a v3.15.1 note.

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx [hunk 6]
```diff
@@ -0,0 +1,38 @@
+22 Mar 2023
+
+### Improvements
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx]: I thinnk this entire block is v3.15.1

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+- Fixed a bug causing liveness checks to fail for Calico Node during upgrades, due to changed field names in TLS secrets.
+- Fixed a bug that prevented [controlPlaneNodeSelector](../../reference/installation/api.mdx#operator.tigera.io/v1.InstallationSpec)
+  from being propagated to Prometheus and Alertmanager.
+- Fixed a bug that caused Intrusion Detection Controller to restart whenever a managed cluster would disconnect.
+- CVE updates, most notably: updating our scratch-based images to UBI 8.7, golang versions 1.18.9 or 1.19.5, ECK 2.5.0, Elasticsearch 7.17.7, Kibana 7.17.7.
+
+### Known issues
+
+- L7 log collection fails to deploy on CIS-hardened clusters. As a result, some cards in the manager UI dashboard will not display any metrics.
+- Upgrading {{prodname}} v3.15.1 on Rancher/RKE from {{prodname}} v3.13.0 currently requires manually terminating the calico-node container for upgrade to proceed.
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx]: should be v3.15.2

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+- Updated TLS configurations to use only TLS1.2.
+- The images for FlexVolume, CSI, Node Driver Registrar are now pulled from our private `quay.io` registry.
+
+### Bug Fixes
+
+- Fixed a bug that caused policies to show in the service graph for even if they were deleted before the selected time range.
+- Fixed an issue where publicly-signed Splunk certificates were not trusted.
+- Fixed a bug that caused policy preview to crash due to a malformed query.
+- Fixed a bug that caused Anomaly Detection jobs to fail if incomplete documents are present in our log storage.
+- Fixed performance issues with listing/resyncing routes when there are many interfaces/routes on the system.  Use kernel-side filtering of routes for a dramatic speed improvement. (Auto-enabled on kernel v4.10+.)
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx]: nit:  "system.  Use" <= 2 spaces.

File: calico-enterprise_versioned_docs/version-3.15/releases.json [hunk 9]
```diff
@@ -1,5 +1,257 @@
 [
   {
+    "title": "v3.15.2",
+    "tigera-operator": {
+      "image": "tigera/operator",
+      "version": "v1.28.10",
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.15/releases.json]: Should be [v1.28.11](https://github.com/tigera/operator/milestone/235)

File: calico-enterprise_versioned_docs/version-3.15/variables.js [hunk 10]
```diff
@@ -1,12 +1,12 @@
...
 
 const variables = {
-  releaseTitle: 'v3.15.1',
+  releaseTitle: 'v3.15.2',
   prodname: 'Calico Enterprise',
   prodnamedash: 'calico-enterprise',
   version: 'v3.15',
-  baseUrl: '/calico-enterprise/latest',
-  filesUrl: 'https://downloads.tigera.io/ee/v3.15.1',
+  baseUrl: '/calico-enterprise/3.15',
```
Comment (ctauchen) [inline calico-enterprise_versioned_docs/version-3.15/variables.js:8]: s/3.15/latest

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx (hunk 5 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx]: The first one is from your post on eng-eng. It sounded like an improvement, not a bug so I put it there. The second one is from 3.15.1, but Matt said patch RN are cumulative, not discrete.

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx (hunk 6 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx]: Yes, by design.

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx (hunk 7 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx]: Fixed.

File: calico-enterprise_versioned_docs/version-3.15/releases.json (hunk 9 above)
Comment (bmckercher123) [reply calico-enterprise_versioned_docs/version-3.15/releases.json]: Fixed.

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx [hunk 11]
```diff
@@ -0,0 +1,38 @@
+22 Mar 2023
+
+### Improvements
+
+- Added tolerations to CSI Driver so it can be scheduled on all nodes.
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx]: This should be a bug fix

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+- Added graceful shutdown to Typha improves the "happy case" for upgrade, ensuring that shutting down a Typha instance doesn't cause a thundering herd of `calico-node` subcomponent restarts. Instead, clients are disconnected slowly over a configured interval, thus limiting the rate of `calico-node` subcomponent restarts.
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx]: This should be deleted.

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx [hunk 13]
```diff
@@ -0,0 +1,38 @@
...
+  - Combining graceful restart with new configuration exposed by the operator allows for the Typha deployment's `maxSurge`
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx]: This should be deleted.

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx [hunk 14]
```diff
@@ -0,0 +1,38 @@
...
+  and `maxUnavailable` settings to be tuned. Setting `maxSurge: "100%"` and `maxUnavailable: 0` means that all up-level
+  Typha instances will start and reach `Ready` before their back-level counterpart is then (gracefully) shut down.
+  (For back-compatibility, the default values of `maxSurge` and `maxUnavailable` have been kept as-is in this release
+  but the upgrade guide explains how to set these during an upgrade.)
+  - To mitigate the "unhappy case" where many `calico-node` instances restart at once (for any reason), Typha's handling
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx]: This should be deleted.

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx [hunk 15]
```diff
@@ -0,0 +1,38 @@
...
+  of new connections has been improved to share much of the work between different clients, this dramatically improves
+  performance when dealing with a "thundering herd".
+- Updated TLS configurations to use only TLS1.2.
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx]: This should be deleted.

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx [hunk 16]
```diff
@@ -0,0 +1,38 @@
...
+- The images for FlexVolume, CSI, Node Driver Registrar are now pulled from our private `quay.io` registry.
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx]: This should be deleted.

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx [hunk 17]
```diff
@@ -0,0 +1,38 @@
...
+- Fixed performance issues with listing/resyncing routes when there are many interfaces/routes on the system. Use kernel-side filtering of routes for a dramatic speed improvement. (Auto-enabled on kernel v4.10+.)
+- Fixed an issue where the operator made many meaningless policy updates, resulting in bloated policy details in the UI.
+- Added a missing policy between Guardian and the Apiserver that could cause issues in multi-cluster installations if a global default deny policy is active.
+- Fixed a bug causing Calico Node to restart during upgrades when upgrading from v3.10 and before, due to the  
+  addition of the TPROXYMode setting in the Felix Configuration resource.
+- Fixed a bug causing liveness checks to fail for Calico Node during upgrades, due to changed field names in TLS secrets.
+- Fixed a bug that prevented [controlPlaneNodeSelector](../../reference/installation/api.mdx#operator.tigera.io/v1.InstallationSpec)
+  from being propagated to Prometheus and Alertmanager.
+- Fixed a bug that caused Intrusion Detection Controller to restart whenever a managed cluster would disconnect.
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx]: This should be deleted.

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx [hunk 18]
```diff
@@ -0,0 +1,38 @@
...
+- Fixed a bug that caused policy preview to crash due to a malformed query.
+- Fixed a bug that caused Anomaly Detection jobs to fail if incomplete documents are present in our log storage.
+- Fixed performance issues with listing/resyncing routes when there are many interfaces/routes on the system. Use kernel-side filtering of routes for a dramatic speed improvement. (Auto-enabled on kernel v4.10+.)
+- Fixed an issue where the operator made many meaningless policy updates, resulting in bloated policy details in the UI.
+- Added a missing policy between Guardian and the Apiserver that could cause issues in multi-cluster installations if a global default deny policy is active.
+- Fixed a bug causing Calico Node to restart during upgrades when upgrading from v3.10 and before, due to the  
+  addition of the TPROXYMode setting in the Felix Configuration resource.
+- Fixed a bug causing liveness checks to fail for Calico Node during upgrades, due to changed field names in TLS secrets.
+- Fixed a bug that prevented [controlPlaneNodeSelector](../../reference/installation/api.mdx#operator.tigera.io/v1.InstallationSpec)
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx]: This should be deleted.

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx [hunk 19]
```diff
@@ -0,0 +1,38 @@
...
+- Fixed an issue where publicly-signed Splunk certificates were not trusted.
+- Fixed a bug that caused policy preview to crash due to a malformed query.
+- Fixed a bug that caused Anomaly Detection jobs to fail if incomplete documents are present in our log storage.
+- Fixed performance issues with listing/resyncing routes when there are many interfaces/routes on the system. Use kernel-side filtering of routes for a dramatic speed improvement. (Auto-enabled on kernel v4.10+.)
+- Fixed an issue where the operator made many meaningless policy updates, resulting in bloated policy details in the UI.
+- Added a missing policy between Guardian and the Apiserver that could cause issues in multi-cluster installations if a global default deny policy is active.
+- Fixed a bug causing Calico Node to restart during upgrades when upgrading from v3.10 and before, due to the  
+  addition of the TPROXYMode setting in the Felix Configuration resource.
+- Fixed a bug causing liveness checks to fail for Calico Node during upgrades, due to changed field names in TLS secrets.
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx]: This should be deleted.

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx [hunk 20]
```diff
@@ -0,0 +1,38 @@
...
+### Bug Fixes
+
+- Fixed a bug that caused policies to show in the service graph for even if they were deleted before the selected time range.
+- Fixed an issue where publicly-signed Splunk certificates were not trusted.
+- Fixed a bug that caused policy preview to crash due to a malformed query.
+- Fixed a bug that caused Anomaly Detection jobs to fail if incomplete documents are present in our log storage.
+- Fixed performance issues with listing/resyncing routes when there are many interfaces/routes on the system. Use kernel-side filtering of routes for a dramatic speed improvement. (Auto-enabled on kernel v4.10+.)
+- Fixed an issue where the operator made many meaningless policy updates, resulting in bloated policy details in the UI.
+- Added a missing policy between Guardian and the Apiserver that could cause issues in multi-cluster installations if a global default deny policy is active.
+- Fixed a bug causing Calico Node to restart during upgrades when upgrading from v3.10 and before, due to the
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx]: This should be deleted.

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx [hunk 21]
```diff
@@ -0,0 +1,38 @@
...
+- The images for FlexVolume, CSI, Node Driver Registrar are now pulled from our private `quay.io` registry.
+
+### Bug Fixes
+
+- Fixed a bug that caused policies to show in the service graph for even if they were deleted before the selected time range.
+- Fixed an issue where publicly-signed Splunk certificates were not trusted.
+- Fixed a bug that caused policy preview to crash due to a malformed query.
+- Fixed a bug that caused Anomaly Detection jobs to fail if incomplete documents are present in our log storage.
+- Fixed performance issues with listing/resyncing routes when there are many interfaces/routes on the system. Use kernel-side filtering of routes for a dramatic speed improvement. (Auto-enabled on kernel v4.10+.)
+- Fixed an issue where the operator made many meaningless policy updates, resulting in bloated policy details in the UI.
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx:11]: This should be deleted.

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx [hunk 22]
```diff
@@ -0,0 +1,38 @@
...
+- Added a missing policy between Guardian and the Apiserver that could cause issues in multi-cluster installations if a global default deny policy is active.
+- Fixed a bug causing Calico Node to restart during upgrades when upgrading from v3.10 and before, due to the  
+  addition of the TPROXYMode setting in the Felix Configuration resource.
+- Fixed a bug causing liveness checks to fail for Calico Node during upgrades, due to changed field names in TLS secrets.
+- Fixed a bug that prevented [controlPlaneNodeSelector](../../reference/installation/api.mdx#operator.tigera.io/v1.InstallationSpec)
+  from being propagated to Prometheus and Alertmanager.
+- Fixed a bug that caused Intrusion Detection Controller to restart whenever a managed cluster would disconnect.
+- CVE updates, most notably: updating our scratch-based images to UBI 8.7, golang versions 1.18.9 or 1.19.5, ECK 2.6.1, Elasticsearch 7.17.9, Kibana 7.17.9.
```
Comment (rene-dekker) [inline calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx]: This should be deleted.

The CVE fixes in the current patch include:
- golang v1.19.7.
- ECK 2.6.1, Elasticsearch 7.17.9, Kibana 7.17.9.
- no ubi changes

Comment (rene-dekker) [reply calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx]: And also update dex to latest (currently 2.35.2

File: calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx (hunk 6 above)
Comment (rene-dekker) [reply calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.2-release-notes.mdx]: Have you tried to look at the preview of the release notes?

**Pull request closed without merging**